### PR TITLE
bug fixes and change cards to match changes to Rafa's new cards

### PR DESCRIPTION
**Pull request by jjhuert99**
did "Reformat Code" on HouseViewModel, that's why its has so many minor changes

### CODE REVIEW

**Review by Inasity — APPROVED** [review]
👍

File: app/src/main/java/com/bps/gotwinter2021/ui/overview/OverviewFragment.kt [hunk 1]
```diff
@@ -32,10 +31,9 @@ class OverviewFragment : Fragment() {
         binding.lifecycleOwner = this
         binding.viewModelOverview = viewModel
         binding.backArrow2.setOnClickListener {
-            Log.d("JJJ","It was Pressed: ${findNavController().popBackStack()}")
+            findNavController().popBackStack()
             findNavController().popBackStack()
```
Comment (Rafa311018) [inline app/src/main/java/com/bps/gotwinter2021/ui/overview/OverviewFragment.kt]: you added findNavController().popBackStack() twice

**Review by Inasity — CHANGES_REQUESTED** [review]
I failed to see the double popBackStack in your code

**Review by Rafa311018 — APPROVED** [review]
looks good

**Review by Rafa311018 — CHANGES_REQUESTED** [review]
looks good, only delete one findNavController().popBackStack()

**Review by Inasity — APPROVED** [review]
Looks good now, 👍

**Review by Rafa311018 — APPROVED** [review]
looks good

**Pull request merged**